sdk/src/main/scala/com/horizen/consensus/package.scala

```diff
@@ -1,18 +1,21 @@
 package com.horizen
 
-import java.math.{BigDecimal, BigInteger, MathContext}
-
 import com.google.common.primitives.{Bytes, Ints, Longs}
+import com.horizen.cryptolibprovider.{CryptoLibProvider, FieldElementUtils}
+import com.horizen.poseidonnative.PoseidonHash
 import com.horizen.vrf.VrfOutput
 import scorex.util.ModifierId
 import supertagged.TaggedType
 
+import java.math.{BigDecimal, BigInteger, MathContext}
+
 package object consensus {
   val merkleTreeHashLen: Int = 32
   val sha256HashLen: Int = 32
-  val consensusNonceLength: Int = Longs.BYTES
+  val consensusNonceLength: Seq[Int] = Seq(8, 32)
```

Comment:
I think if we really need this.
Possibly, just for double check.
But in anyway, it is not very clear now from the parameter type how it is used. Let's change the name to something like `consensusNonceAllowedLengths`